ui/cypress.json

```diff
@@ -2,6 +2,9 @@
   "env": {
     "target_url": "http://localhost:3000",
     "username": "admin",
-    "password": "admin"
+    "password": "admin",
```

Comment:
About the commit message; it's `2nd`, not `2rd` :wink: 

Also, could the commit title is a bit long (that's not a big issue, but when possible we try to keep it around 50 characters, see our [GUIDELINES](https://github.com/scality/metalk8s/blob/development/2.4/CONTRIBUTING.md#committing-guidelines)).

I suggest something like:

```
ui/test: add a 2nd E2E test for volume creation

Now check that we can also create a sparseLoopDevice Volume
```